### ISSUE TITLE
new tab/no iframe not working

### ISSUE THREAD

**Issue by elektroschrott** [causefx/Organizr#959]
##### Problem Description:

'no iframe' doesn't work on https://mediathekviewweb.de because of CSP. Any way to integrate this site to organizr?
<hr>

##### Reproduction Steps:

<hr>

#### Errors on screen?  If so paste here:

```

```

**Comment by elektroschrott** [comment]
```
Blockiert von der Inhaltsicherheitsrichtlinie (CSP)

Beim Verbinden mit mediathekviewweb.de trat ein Fehler auf.

Firefox hat diese Webseite daran gehindert, auf diese Weise geladen zu werden, weil die Webseite eine Inhaltsicherheitsrichtlinie (Content Security Policy) hat, die dies nicht erlaubt.
```

**Comment by elektroschrott** [comment]
![grafik](https://user-images.githubusercontent.com/34006335/47482243-4a3ac800-d836-11e8-94cc-d0c09e62326c.png)

**Comment by elektroschrott** [comment]
btw: this issue happens only, if a URL is set as 'standard'...: 'no iframe' doesnt't work...

**Comment by causefx** [comment]
are you running v1 or v2?

**Comment by Weissnimma** [comment]
Yes of course! I use the docker lsiocommunity/organizr... Should I try another one? How to get V2?

**Comment by causefx** [comment]
yes. try this:
organizrtools/organizr-v2

**Comment by Weissnimma** [comment]
Ok. Works with 'new windows'. But I can't start my 'Shell in a Box' with IP:4200... How should I do this avoiding a new windows?

**Comment by Weissnimma** [comment]
And is there a way to disable the splash screen (or make it simple like v1)...?

**Comment by Weissnimma** [comment]
If I tell to open 'IP:4200', it gives a error opening 'IP:4321/#OMV-Bash' (4321 is the port of organizr2)

**Comment by causefx** [comment]
what do you mean you cant start SIAB?

if you are manually typing in that ip and port into browser, organizr should have zero effect on it.

for splash, dont enable spash for any tabs, that will disable it

**Comment by Weissnimma** [comment]
Works with deacitivating spash, thanx!
I can opten SIAB (port 4200) manually, but not via organizr2. Was no problem on V1.8 iframe.

**Comment by Weissnimma** [comment]
The main frame stays empty when trying to open SIAB, the URL goes to http://192.168.178.80:4321/#OMV-Bash

**Comment by causefx** [comment]
What URL did you type in exactly in tab settings?

**Comment by Weissnimma** [comment]
192.168.178.80:4200

**Comment by causefx** [comment]
add in `http:// `

**Comment by Weissnimma** [comment]
same problem... opens 'http://192.168.178.80:4321/#OMV-Bash' and should open http://192.168.178.80:4200...

**Comment by Weissnimma** [comment]
I continue tomorrow... its midnight here :-)

**Comment by causefx** [comment]
no worries, the url in browser is just showing the hash of the current tab you are going to.
